Cargo.toml

```diff
@@ -16,31 +16,38 @@ rust-version = "1.61.0"
 [lib]
 name = "chrono"
 
-[features]
-# Don't forget to adjust `ALL_NON_EXCLUSIVE_FEATURES` in CI scripts when adding a feature or an optional dependency.
-default = ["clock", "std"]
-alloc = []
-libc = []
-winapi = ["windows-targets"]
-std = ["alloc"]
-clock = ["winapi", "iana-time-zone", "android-tzdata", "now"]
+[features] # Don't forget to adjust `ALL_NON_EXCLUSIVE_FEATURES` in CI scripts when adding a feature or an optional dependency.
```

Comment:
This commit has a lot going on, can you split it in smaller pieces? For example, one for introducing the use of `dep:`, one for cleaning up the `clock`-related stuff, other stuff...

Also I think I'd prefer to have single-line comments on the same line as the feature, and keep lists of features on one line.

Reply:
Done. Second try on making the commits somewhat reasonable :smile:.

> Also I think I'd prefer to have single-line comments on the same line as the feature, and keep lists of features on one line.

Sorry, I didn't write down what I was trying to do. libs.rs can show documentation for crate features but I am not sure of the details. Having a comment above the feature works for sure.

We have a CI run that checks the format of cargo.toml. I think that is quite useless personally. It wants the list of features for `clock` to be on multiple lines.